knowledge_repo/converters/ipynb.py

```diff
@@ -1,11 +1,23 @@
 import os
+import nbconvert
 
 from ..converter import KnowledgePostConverter
 from .._version import __optional_dependencies__
 
 TEMPLATE = '''
 {%- extends 'markdown.tpl' -%}
 
+{%- block data_javascript scoped %}
+{% set div_id = uuid4() %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_javascript {{ extra_class }}">
+<script type="text/javascript">
```

Comment:
The `element` variable I'm talking about is: on line 14 (the one under this annotation as it turns out... whoops!
`var element = $('#{{ div_id }}');`